.github/workflows/pr.yaml

```diff
@@ -131,10 +131,11 @@ jobs:
           sudo lxc cluster enable local
           sudo lxc config set user.show_permissions=true
 
-      - name: Create a custom image
+      - name: Create and download custom images
         shell: bash
         run: |
           set -x
+          curl -L -o ubuntu-minimal.qcow2 https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img &
```

Comment:
this also needs to be added to coverage.yaml

Reply:
yes, thanks for catching that :+1: I wonder if it's useful to create reusable actions for these jobs that's almost identical so that it's easier to maintain in the future?